src/sentry/search/events/fields.py

```diff
@@ -1791,22 +1821,28 @@ def is_accessible(self, acl=None):
             ],
             default_result_type="number",
         ),
-        # Currently only used by trace meta so we can count event types which is why this only accepts strings
+        # The calculated arg will cast the string value according to the value in the column
         Function(
             "count_if",
             required_args=[
-                ColumnNoLookup("column", allowed_columns=["event.type", "http.status_code"]),
+                FunctionArg("column"),
```

Comment:
Why is this a `FunctionArg` instead of `ColumnNoLookup` now? This removes the validation that it's a real column right? Is this to support tags as well?

Reply:
Exactly, this is cause I want to support tags like `http.status_code` i'll add a comment :+1: